onnxruntime/core/providers/cpu/math/gemm.cc

```diff
@@ -11,4 +11,10 @@ ONNX_CPU_OPERATOR_VERSIONED_KERNEL(
     9,
     KernelDefBuilder().TypeConstraint("T", DataTypeImpl::GetTensorType<float>()),
     Gemm<float>);
-}
+
+ONNX_CPU_OPERATOR_KERNEL(
+    Gemm,
```

Comment:
Would be good to have a comment saying bias was made optional

Reply:
sure.

Reply:
Added a comment.